### PR TITLE
On Premise Deployment Document Updates

### PR DESCRIPTION
**Pull request by resmaa**
Revisit on On-premise Page.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1032#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 5465f39c21f5b89ce42164a72db397f4ed86dd3c | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1032/librarium_build/1602274571069689856) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/12-enterprise-version.md [hunk 1]
```diff
@@ -25,3 +25,35 @@ Highly available multi-node installation typically for production purposes.
 
 Palette provides the platform installer to enable quick start mode and enterprise mode. 
 To obtain the platform installer, please contact our Sales team by completing [the form here](https://www.spectrocloud.com/contact/).
+
+<br />
+
+
+## Important Resources
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version.md:32]:
```suggestion
# Resources 
```
I think we agreed to label this section as Resources in our team meeting.

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md [hunk 2]
```diff
@@ -66,19 +100,57 @@ The following points needs to be taken care while creating the Tags:
 
 </InfoBox>
 
-### Permissions
 
-The following permissions are required for the account used to install the platform:
+## Privilages
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/00-on-prem-system-requirements.md:104]:
```suggestion
## Privileges 
```
Small misspelling

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md [hunk 3]
```diff
@@ -66,19 +100,57 @@ The following points needs to be taken care while creating the Tags:
 
 </InfoBox>
 
-### Permissions
 
-The following permissions are required for the account used to install the platform:
+## Privilages 
+
+**Last Update**: August 18, 2022
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/00-on-prem-system-requirements.md:106]:
```suggestion
```
We should remove this. We don't specify anywhere else in the docs when a document was updated.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1032#pullrequestreview-1213693226" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1032#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/00-on-prem-system-requirements.md (hunk 2 above)
Comment (resmaa) [reply content/docs/12-enterprise-version/00-on-prem-system-requirements.md:104]: thanks

**Pull request merged**